hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/ServerCall.java

```diff
@@ -431,31 +436,42 @@ public boolean isClientCellBlockSupported() {
 
   @Override
   public long getResponseCellSize() {
-    return responseCellSize;
+    return responseCellSize.get();
   }
 
   @Override
-  public void incrementResponseCellSize(long cellSize) {
```

Comment:
Are these methods removed?

Reply:
The logic has been moved to RpcCallContext#addResultSize now